lib/iris/fileformats/pp.py

```diff
@@ -1403,9 +1438,11 @@ def save(self, file_handle):
         # the data itself
         if lb[self.HEADER_DICT['lbpack'][0]] == 0:
             data.tofile(pp_file)
+        elif lb[self.HEADER_DICT['lbpack'][0]] == 1:
+            pp_file.write(packed_data)
         else:
             msg = 'Writing packed pp data with lbpack of {} ' \
-                'is not supported.'.format(lb[self.HEADER_DICT['lbpack'][0]])
+                  'is not supported.'.format(lb[self.HEADER_DICT['lbpack'][0]])
             raise NotImplementedError(msg)
```

Comment:
To me, ImportError would suggest that a package that packed data with lbpacks that aren't 0 or 1 exists. As far as I'm aware, this doesn't currently exist. 
It appears that you should only get ImportErrors  when you've tried to import a package (https://docs.python.org/2/library/exceptions.html#exceptions.ImportError)

NotImplementedError was what we had before when we didn't even have wgdos packing, so surely we have the same situation here?